### PR TITLE
Math magicians: tests

### PR DESCRIPTION
**Pull request by mrparkersson**
Pull request
Amo Parker Prince

What does this PR do?

Designing Calculator setup template and adding test files in the component directory

Description of the task to be completed
A tests using Jest and React test library 

Used React in writing JSX and adding styles to the template

created test.MD files that have been ignored
How should it be tested
By running npm test in the terminal.

clone the repository
git clone https://github.com/mrparkersson/Calculator
navigate to the folder
cd mycalculator
checkout the feature

### CODE REVIEW

File: src/Components/tests/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import operate from '../../Logic/operate';
+
+describe('', () => {
+  test('', () => {
+    expect(operate(1, 2, '+')).toBe('3');
+  });
+});
```
Comment (omar-labana) [inline src/Components/tests/operate.test.js]: - [ ] Kindly, cover all functions found in the `operate` module with proper tests, I can see only 1 operation being tested. Also, I would suggest adding tests for negative cases.

File: src/Components/tests/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  });
+  test('Testing with + buttton', () => {
+    const obj = {
+      total: 2,
+      next: 4,
+      operation: '-',
+    };
+    const results = calculate(obj, '-');
+    expect(parseInt(results.total)).toBe(-2);
+  });
```
Comment (omar-labana) [inline src/Components/tests/calculate.test.js]: - [ ] Kindly, cover all possible operations found in the `calculate` module with proper tests(only AC, +, - operations are being tested), Also, I would suggest adding tests for negative cases.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @mrparkersson, 

Excellent work so far! 👌
You have implemented almost all the required tests. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
According to this milestone's requirements:
- [ ] Jest snapshots should be used to test all of the react components(I can only find one test covering the `Navbar` component), kindly, cover the rest of the components. 


- [ ] _Please, check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hi @mrparkersson   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
